src/calendar/tests/CalendarHighlight.spec.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react/addons';
+import CalendarHighlight from '../CalendarHighlight';
+
+const TestUtils = React.addons.TestUtils;
+
+describe('The CalendarHighlight Component', function () {
+  beforeEach(function () {
+    this.spyCx = spyOn(CalendarHighlight.prototype.__reactAutoBindMap, 'cx').and.returnValue('my-class');
```

Comment:
In this specific example, the class name would be: 'null__CalendarHighlight null__CalendarHighlight--test'. This output is simpler than some I have seen.

I reckon that if we test if the class name contains 'CalendarHighlight', we would be getting roughly the same testing expectations. We would be moving the contract uncertainty away from React into our BEM implementation that we do control. 

Just let me know if that works for you and I will refactor.